scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-2a-sbd-kvm.yaml

```diff
@@ -89,7 +89,7 @@ proposals:
     ha:
       storage:
         shared:
-          device: 10.162.26.129:/var/qa2/ha-database
+          device: ##shared_nfs_for_database##
```

Comment:
it is currently hardcoded in the jenkins job scenario file. To be clarified and unified by the card in team green about cleaning up the qa jobs and unifying them with JJB